webapp/channels/src/i18n/en.json

```diff
@@ -5430,6 +5430,10 @@
   "user.settings.notifications.header": "Notifications",
   "user.settings.notifications.icon": "Notification Settings Icon",
   "user.settings.notifications.info": "Desktop notifications are available on Edge, Firefox, Safari, Chrome and Mattermost Desktop Apps.",
+  "user.settings.notifications.keywordsWithHighlight.extraInfo": "These keywords will be shown to you with a highlight when anyone sends a message that includes them.",
+  "user.settings.notifications.keywordsWithHighlight.inputTitle": "Enter non case-sensitive keywords, press Tab or use commas to separate them:",
+  "user.settings.notifications.keywordsWithHighlight.none": "None",
+  "user.settings.notifications.keywordsWithHighlight.title": "Keywords That Get Highlighted (without notifications)",
```

Comment:
@abhijit-singh seems there is inconsistency in casing with this title. Should the words within the parenthesis also be in capitals?